lom/src/plugins/sonic/plugin/detection/link/linkcrc/link_crc.go

```diff
@@ -0,0 +1,248 @@
+/* This file contains logic for Link CRC anomaly detection. It detects CRC anomlies for all eligible interfaces on the device.*/
+package linkcrc
```

Comment:
Add a Readme

Explain the algorithm used

Reply:
Done. Added.